### ISSUE TITLE
Regression in Wiki Documentation in SC2154

### ISSUE THREAD

**Issue by kumar-ish** [koalaman/shellcheck#2605]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2154
- My shellcheck version (`shellcheck --version` or "online"): online
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

From the documentation for [SC2154](https://github.com/koalaman/shellcheck/wiki/SC2154/49c0ddbdf94e26e257d6692764e41c731aa7fd94) as of the current revision, this should be giving a warning for SC2154:
```sh

#!/bin/bash
target="world"
echo "hello $target"  # misspelled

```

#### Here's what shellcheck currently says:
```
No issues detected!
```

#### Here's what I wanted or expected to see:
```
...
        ^-- [SC2154](https://www.shellcheck.net/wiki/SC2154) (warning): target is referenced but not assigned (did you mean '<CORRECT_VARIABLE_NAME>'?).
```

It looks like there was a regression in the Wiki documentation, [here](https://github.com/koalaman/shellcheck/wiki/SC2154/_compare/7341de17e49560fa0a308e42a6383d41c63ce06e...49c0ddbdf94e26e257d6692764e41c731aa7fd94). Because this is supposed to be giving examples of _problematic_ code that would give a warning for SC2154, the word "target" should actually be misspelled (as it was before), such that it gives a warning.

**Comment by kumar-ish** [comment]
Oops, I didn't realise that -- I might add descriptions for some of the missing warnings in the documentation then!
